master/pkg/model/user.go

```diff
@@ -316,3 +316,25 @@ func HashPassword(password string) (string, error) {
 	}
 	return string(passwordHash), nil
 }
+
+// TokenID is the type for user token IDs.
+type TokenID int
+
+// LongLivedToken corresponds to a row in the "long_lived_tokens" DB table.
```

Comment:
I'm getting more convinced that a refactor to use `AccessToken` instead of `LongLivedToken` might be clearer, but that's definitely another thing that we should run by product